src/Chirp.Web/Pages/UserTimeline.cshtml

```diff
@@ -44,6 +44,9 @@
                 </li>
             }
         </ul>
+        <div style="display: flex; justify-content: center;">
+            <button><a href="/@(Model.authornextpage)?page=@(Model.page+1)">Next Page</a></button>
```

Comment:
use `<button><a href="/@routeName?page=@(Model.page+1)">Next Page</a></button>`

Instead, removes the need for the string in the Model